SmartDeviceLink/SDLDateTime.h

```diff
@@ -0,0 +1,73 @@
+//  SDLDateTime.h
+//
+
+#import "SDLRPCStruct.h"
+
+@interface SDLDateTime : SDLRPCStruct
+
+- (instancetype)initWithHour:(UInt8)hour minute:(UInt8)minute second:(UInt8)second millisecond:(UInt16)millisecond day:(UInt8)day month:(UInt8)month year:(UInt16)year timezoneMinuteOffset:(UInt8)timezoneMinuteOffset timezoneHourOffset:(int)timezoneHourOffset;
```

Comment:
I'd like to see a date time only for day / month / year, one without the timezone offsets, maybe one that takes an `NSDate`, or an `NSTimeZone`, and one for only hour / minute / second / millisecond, or hour / minute / second?